src/stepper/advance.f90

```diff
@@ -288,19 +301,21 @@ subroutine adapt(bstep, t, dt)
             buf(3) = umax
             buf(4) = vmax
             buf(5) = wmax
-            buf(6) = uszrms
-            buf(7) = ussrms
+            buf(6) = usggmax
+            buf(7) = lsggmax
+            buf(8) = uszrms
+            buf(9) = ussrms
```

Comment:
It is up to you, but I'm not sure we'll every look at this quantity, but there is no hard in keeping it

Reply:
I can easily remove all of this.